setuptools/tests/config/test_pyprojecttoml_dynamic_deps.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+
+from setuptools.config.pyprojecttoml import apply_configuration
+from setuptools.dist import Distribution
+
+
+def test_dynamic_dependencies(tmp_path):
+    (tmp_path / "requirements.txt").write_text("six\n  # comment\n")
+    pyproject = (tmp_path / "pyproject.toml")
+    pyproject.write_text("""
+[project]
+name = "myproj"
+version = "1.0"
+dynamic = ["dependencies"]
+
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.dynamic.dependencies]
+file = ["requirements.txt"]
+""")
```

Comment:
This is a minor comment :)

In other test files, setuptools uses `from .textwrap import DALS` to deal with the indentation. I think `from inspect import cleandoc` is also an alternative built-in Python3 stdlib.

Reply:
Yeah, I saw DALS elsewhere, but I also noticed not all tests use it (though now that I think of it, TOML doesn't strictly speaking need to be DALS-ed). Fixed, nevertheless!